deploy/helm/flightctl/templates/flightctl-db-deployment.yaml

```diff
@@ -68,9 +68,6 @@ spec:
             requests:
               cpu: {{ .Values.db.resources.requests.cpu}}
               memory: {{ .Values.db.resources.requests.memory}}
-            limits:
-              cpu: {{ .Values.db.resources.limits.cpu}}
-              memory: {{ .Values.db.resources.limits.memory}}
```

Comment:
just curios why remove this?

Reply:
It's an ACM requirement - see ACM-16348 - the idea is to deal with resource issues rather than have the system killing pods.